test/support/govspeak_validation_test_helper.rb

```diff
@@ -1,12 +1,17 @@
 module GovspeakValidationTestHelper
-  def should_protect_against_xss_and_content_attacks_on(*attributes)
+  def should_protect_against_xss_and_content_attacks_on(*attributes, supplied_factory_name)
```

Comment:
why have changes been made to this file?

Reply:
The method previously made an assumption that you wanted to run the XSS tests against the factory whose name matched the class being tested (e.g. the `:world_location_news` factory would be used for the `WorldLocationNews` model).  However we want to test against the `:world_location_news_with_stubbed_slug` factory instead, so can't use this assumption anymore.

We shifted this to always be the first argument to the method as it's no longer optional and would've had to be utilised by appending `supplied_factory_name: :foo` to the method calls, because of the usage of the splat operator prior to this.  This change means we can just add it as the first argument.